include/gatekeeper_gt.h

```diff
@@ -21,15 +21,34 @@
 
 #include <stdint.h>
 
+#include <rte_ip.h>
+#include <rte_tcp.h>
+#include <rte_udp.h>
 #include <rte_atomic.h>
 
+#include "gatekeeper_config.h"
+
+struct gt_packet_headers {
+	uint16_t outer_ip_ver;
+	uint16_t inner_ip_ver;
+	uint8_t l4_proto;
+
```

Comment:
Add priority after `l4_proto` as a `uint8_t`. It won't add more space, and it's a relevant information for policies.